### DISCUSSION TITLE
Vivado中的 Xsim 與 Modelsim 模擬結果不一樣

### DISCUSSION THREAD

**Discussion by yushuodev** [bol-edu/HLS-SOC-Discussions#113] [Q&A]
![photo_2023-10-18_15-42-41](https://github.com/bol-edu/HLS-SOC-Discussions/assets/77037575/6d206754-e74b-4049-a731-1a69004a6068)
![photo_2023-10-18_15-42-36](https://github.com/bol-edu/HLS-SOC-Discussions/assets/77037575/1b624f6e-70fc-46e9-a2d4-574e9e5ce99d)
如上圖我在Vivado中無法跑模擬執行第二次時會出錯，但我在其他Simulation軟體中，Modelsim上是沒問題的
想詢問大家是否有遇到不同軟體simulation不同的狀況，並且如何解決的，感謝

**Comment by boledulab** [comment]
You may need to provide your testbench, or provide more information.
